packages/@rnw-scripts/doxysaurus/src/transformer.ts

```diff
@@ -0,0 +1,327 @@
+        const sectionKind = sectionDef.$.kind;
+        const sectionName = knownSections.get(sectionKind);
+
+        if (typeof sectionName === 'undefined') {
+          throw new Error(`Unknown section kind ${sectionKind}`);
+        }
+
+        if (sectionName === '<not visible>') {
+          continue;
+        }
+
+        let section: DocSection;
+        if (sectionName === '<user defined>') {
+          section = new DocSection();
+          section.name = sectionDef.header[0]._;
+          visibleSections[section.name] = section;
+        } else {
+          section = visibleSections[sectionName];
+          if (typeof section === 'undefined') {
+            section = new DocSection();
+            section.name = sectionName;
+            visibleSections[sectionName] = section;
+          }
+        }
+
+        const memberOverloads = new Map<string, DocMemberOverload>();
+
+        for (const memberDef of sectionDef.memberdef) {
+          const memberName = memberDef.name[0]._;
+          const member = new DocMember();
+          member.line = Number(memberDef.location[0].$.line);
+          memberToDoxMember.set(member, memberDef);
+          member.name = memberName;
+          const overloadName =
+            memberName === compound.name
+              ? '(constructor)'
+              : memberName === '~' + compound.name
+              ? '(destructor)'
+              : memberName === 'operator='
+              ? 'assignment operator='
+              : memberName === 'operator=='
+              ? 'equal operator=='
+              : memberName === 'operator!='
+              ? 'not equal operator!='
+              : memberName === 'operator[]'
+              ? 'subscript operator[]'
+              : memberName;
+          let memberOverload = compoundMemberOverloads.get(overloadName);
+          if (typeof memberOverload === 'undefined') {
+            memberOverload = new DocMemberOverload();
+            memberOverloadToCompound.set(memberOverload, compound);
+            memberOverload.name = overloadName;
+            memberOverload.line = member.line;
+            compoundMemberOverloads.set(overloadName, memberOverload);
+          }
+          if (!memberOverloads.has(overloadName)) {
+            memberOverloads.set(overloadName, memberOverload);
+            section.memberOverloads.push(memberOverload);
+          }
+          memberOverload.members.push(member);
+          memberOverload.line = Math.min(memberOverload.line, member.line);
+
+          doxMemberIdToMemberOverload.set(memberDef.$.id, memberOverload);
+
+          if (section.line === 0) {
+            section.line = member.line;
+          } else {
+            section.line = Math.min(section.line, member.line);
+          }
+        }
+
+        section.memberOverloads.sort((a, b) => a.line - b.line);
+      }
+
+      const sections: DocSection[] = Object.values(visibleSections);
+      // Put deprecated section to the end
+      sections.forEach(
```

Comment:
>sections.forEach( [](start = 6, length = 17)

Why not use regular foreach eg: for(...in...) ?

Reply:
you mean `for ( xxx of yyy )` :D

Reply:
Good catch. I have missed this place.